Foundation/Process.swift

```diff
@@ -387,6 +387,30 @@ open class Process: NSObject {
             posix(posix_spawn_file_actions_addclose(&fileActions, fd))
         }
 
+        let currentDirectoryURL = URL(fileURLWithPath: currentDirectoryPath)
+        let changeDirectorySucceeded = currentDirectoryURL.withUnsafeFileSystemRepresentation { rawPath -> Bool in
+            guard let rawPath = rawPath else {
+                return false
+            }
+
+            // posix_spawn doesn't support passing a custom working directory, so resort to chdir.
+            return chdir(rawPath) == 0
```

Comment:
Am I missing something, or don't we have to reset this value after we're done with the launch?